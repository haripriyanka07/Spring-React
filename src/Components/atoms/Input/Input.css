input[type=text], select, input[type=number], textarea{
    width: 100%;
    padding: 12px 20px;
    margin: 8px 0;
    display: inline-block;
    border: 1px solid #ffe2b3;
    border-radius: 4px;
    box-sizing: border-box;
  }