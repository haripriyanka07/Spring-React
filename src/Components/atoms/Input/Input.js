import './Input.css';

const Input = (props) => {
    return (
        <input type={props.type} placeholder={props.placeholder} onChange={(e) => props.onChange(e.target.value)} value={props.value} />
    );
}
export default Input;
