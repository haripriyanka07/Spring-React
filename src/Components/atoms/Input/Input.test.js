import React from 'react';
import renderer from 'react-test-renderer';
import Input from './Input';

it('test the input', () => {
    const renderedComponent = renderer.create(<Input type="text" placeholder="Enter..." value="New Text Input" />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
})