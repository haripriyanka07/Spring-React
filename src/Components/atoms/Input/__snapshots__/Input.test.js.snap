// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the input 1`] = `
<input
  onChange={[Function]}
  placeholder="Enter..."
  type="text"
  value="New Text Input"
/>
`;
